scripts/install/jellyfin.sh

```diff
@@ -88,7 +123,7 @@ CONFIG
 cat > /etc/jellyfin/network.xml <<- CONFIG
 <?xml version="1.0" encoding="utf-8"?>
 <NetworkConfiguration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
-  <RequireHttps>true</RequireHttps>
+  <RequireHttps>false</RequireHttps>
```

Comment:
The original spirit of this script created self-signed certs for use with Jellyfin, which is why this was left enabled.